torch/_inductor/output_code.py

```diff
@@ -252,6 +251,35 @@ def maybe_realign_inputs(
             compiled_graph.current_callable = new_callable
 
 
+class CompiledFxGraphConstants:
+    """Wrapper class that gets constants from a compiled fx graph"""
```

Comment:
It would be nice if this parent class explained the subclass inheritance situation and when this one versus the other got used